entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+
+if [[ "${DB_TYPE}" == "postgres" ]]; then
```

Comment:
Aren't we already depending on `db` service? I think running `db` with the `-d` option is pretty common and it'd speed setting up environment quite a bit.

Reply:
Will remove.